meta-resin-common/recipes-support/resin-mounts/resin-mounts.bb

```diff
@@ -12,14 +12,15 @@ SRC_URI = " \
     file://etc-systemd-timesyncd.conf.mount \
```

Comment:
The git log says:

> The second change this introduces is the renaming from mnt-(label).service to
> label.service.

I think this should be something like:

> The second change this introduces is the renaming from mnt-(label).service to
> resin-(label).service.

Reply:
I will reword. Your suggestion is not correct either because label is `resin-state` for example. Not `state`.